configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
@uartie if I am adding a patch I am hoping the gtest is already present and I, as a developer, should make sure that such patch was tested with gtest (which can be done by QA through an automated process).  If I want to add a feature, part of my feature is to add gtest unittests, I should have discussed my feature with others before trying to add it.